npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "perseo",
-  "version": "0.4.1-next",
+  "version": "0.5.0-next",
```

Comment:
We are not currently using the shrinkwrap in the PEP proxy (maybe we should), so this problem won't reproduce there. I'll modify the release script to fix this.